Composer/packages/client/src/pages/design/ManifestEditor.tsx

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core';
+import formatMessage from 'format-message';
+import ErrorBoundary from 'react-error-boundary';
+import React from 'react';
+import { LoadingTimeout } from '@bfc/adaptive-form/lib/components/LoadingTimeout';
+import { FieldLabel } from '@bfc/adaptive-form/lib/components/FieldLabel';
+import ErrorInfo from '@bfc/adaptive-form/lib/components/ErrorInfo';
+import { FontSizes } from '@uifabric/fluent-theme';
+import { Link } from 'office-ui-fabric-react/lib/Link';
+import { FontWeights } from '@uifabric/styling';
+import { DetailsList, DetailsListLayoutMode, SelectionMode } from 'office-ui-fabric-react/lib/DetailsList';
+import get from 'lodash/get';
+
+import { SkillInfo } from '../../recoilModel';
+
+import { formEditor } from './styles';
+
+const styles = {
+  errorLoading: css`
+    padding: 18px;
+  `,
+
+  banner: css`
+    border-bottom: 1px solid #c8c6c4;
+    padding: 0 18px;
+    margin-bottom: 0px;
+  `,
+
+  title: css`
+    font-size: ${FontSizes.size20};
+    font-weight: ${FontWeights.semibold};
+    margin: '5px 0 7px -9px';
+  `,
+
+  subtitle: css`
+    height: 15px;
+    line-height: 15px;
+    font-size: ${FontSizes.size12};
+    color: #4f4f4f;
+    margin: -7px 0 7px;
+  `,
+
+  description: css`
+    margin-top: 0;
+    margin-bottom: 10px;
+    white-space: pre-line;
+    font-size: ${FontSizes.size12};
+  `,
+
+  body: css`
+    padding: 0 18px;
+    font-size: ${FontSizes.size12};
+    section {
+      padding-top: 20px;
+    }
+    label {
+      color: #585756;
+      font-size: ${FontSizes.size12};
+    }
+    .ms-DetailsHeader {
```

Comment:
Is overriding a fluent class the only way we can get this to work? Are margin or padding props not supported on the DetailList component?

Reply:
It seems cannot directly set DetailsHeader's margin on <DetailList> props. then I tried on customize `onRenderDetailsHeader` 
```
(props, defaultRender) => {
{defaultRender({
     ...props,
     styles: {
         root: {
              paddingTop: 0;
         }
     }
})}
}
```
but also not work.